osu.Game/Rulesets/Mods/ModTouchDevice.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Localisation;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public class ModTouchDevice : Mod, IApplicableMod
```

Comment:
It'd be nice to get an icon in for this, even if it's a placeholder. Reasoning is that I think some may see the "TD" appear and be quite confused at first. If it has a recognisable icon it will likely help users understand what's going on.

Reply:
I've applied the most easily usable placeholder in https://github.com/ppy/osu/pull/25348/commits/204cd541e2b8876089c619abd94e97baafb97b8b but I dunno about the end result...

![image](https://github.com/ppy/osu/assets/20418176/ae181f56-92f0-44f5-a269-61125a2dfda9)

- old style icon (https://github.com/ppy/osu/pull/22175 tried to add new ones but also did a bunch of other strange changes we didn't like)
- bad contrast

Mod icons continue to be a shambles. We really need to sort this out and unify the display with web too but that's at least partially a designer job and I dunno when that is ever gonna happen at this rate. Unless I activate the "programmer art" protocol but I'm not sure anybody wants *that* to happen.

Reply:
I think this is better than TD acronym, although the grey colour is pretty off-putting. Could we easily change the system mods colour at this point, or is it more involved?

Reply:
Would need to do a sweep over all mod displays so definitely not something I'd be wanting to put in this diff. Also requires input as to which colours to choose.

Reply:
Would something like this work?

![1699266040](https://github.com/ppy/osu/assets/20418176/6aac446a-f387-4df1-bcb3-cd252b1f2dda)

Based on the "no mod" icon from osu-web:

![image](https://github.com/ppy/osu/assets/20418176/caec6f00-b2db-47dd-b982-8af946c59c30)

Reply:
definitely better

Reply:
PR'd separately as https://github.com/ppy/osu/pull/25377.